src/components/modal/iframe/modal.jsx

```diff
@@ -0,0 +1,39 @@
+var classNames = require('classnames');
+var React = require('react');
+
+var Modal = require('../base/modal.jsx');
+
+require('./modal.scss');
+
+Modal.setAppElement(document.getElementById('view'));
+
+var IframeModal = React.createClass({
+    propTypes: {
+        isOpen: React.PropTypes.bool,
+        onRequestClose: React.PropTypes.func,
+        className: React.PropTypes.string,
+        componentRef: React.PropTypes.string,
```

Comment:
String refs are deprecated, and for this usage, we most likely would always want to use a function ref anyway, so I think this should change to `React.PropTypes.func`.